storage/database/db_manager_stakinginfo_test.go

```diff
@@ -25,7 +25,7 @@ func TestDatabaseManager_StakingInfo(t *testing.T) {
 	dbm := dbManagers[0]
 
 	key := uint64(1234)
-	value := []byte("{\"BlockNum\":2880,\"CouncilNodeAddrs\":[\"0x159ae5ccda31b77475c64d88d4499c86f77b7ecc\",\"0x181deb121304b0430d99328ff1a9122df9f09d7f\",\"0x324ec8f2681cd73642cc55057970540a1f4393e0\",\"0x11191029025d3fcd21001746f949b25c6e8435cc\"],\"CouncilStakingAddrs\":[\"0x70e051c46ea76b9af9977407bb32192319907f9e\",\"0xe4a0c3821a2711758306ed57c2f4900aa9ddbb3d\",\"0xf3ba3a33b3bf7cf2085890315b41cc788770feb3\",\"0x9285a85777d0ae7e12bee3ffd7842908b2295f45\"],\"CouncilRewardAddrs\":[\"0xd155d4277c99fa837c54a37a40a383f71a3d082a\",\"0x2b8cc0ca62537fa5e49dce197acc8a15d3c5d4a8\",\"0x7d892f470ecde693f52588dd0cfe46c3d26b6219\",\"0xa0f7354a0cef878246820b6caa19d2bdef74a0cc\"],\"KIRAddr\":\"0x673003e5f9a852d3dc85b83d16ef62d45497fb96\",\"PoCAddr\":\"0x576dc0c2afeb1661da3cf53a60e76dd4e32c7ab1\",\"UseGini\":false,\"Gini\":-1,\"CouncilStakingAmounts\":[5000000,5000000,5000000,5000000]}")
+	value := []byte("{\"BlockNum\":2880,\"CouncilNodeAddrs\":[\"0x159ae5ccda31b77475c64d88d4499c86f77b7ecc\",\"0x181deb121304b0430d99328ff1a9122df9f09d7f\",\"0x324ec8f2681cd73642cc55057970540a1f4393e0\",\"0x11191029025d3fcd21001746f949b25c6e8435cc\"],\"CouncilStakingAddrs\":[\"0x70e051c46ea76b9af9977407bb32192319907f9e\",\"0xe4a0c3821a2711758306ed57c2f4900aa9ddbb3d\",\"0xf3ba3a33b3bf7cf2085890315b41cc788770feb3\",\"0x9285a85777d0ae7e12bee3ffd7842908b2295f45\"],\"CouncilRewardAddrs\":[\"0xd155d4277c99fa837c54a37a40a383f71a3d082a\",\"0x2b8cc0ca62537fa5e49dce197acc8a15d3c5d4a8\",\"0x7d892f470ecde693f52588dd0cfe46c3d26b6219\",\"0xa0f7354a0cef878246820b6caa19d2bdef74a0cc\"],\"KIRAddr\":\"0x673003e5f9a852d3dc85b83d16ef62d45497fb96\",\"KGFAddr\":\"0x576dc0c2afeb1661da3cf53a60e76dd4e32c7ab1\",\"UseGini\":false,\"Gini\":-1,\"CouncilStakingAmounts\":[5000000,5000000,5000000,5000000]}")
```

Comment:
StakingInfo is stored in the marshalled JSON format using `addStakingInfoToDB` as you can see in this value. If you change the field name of `PoCAddr`, the old data stored in DB is not compatible with a new version.

Reply:
Um, okay. Then i cannnot rename PoC to KGF in StakingInfo struct.
Then should i modify `func (api *PublicGovernanceAPI) GetStakingInfo(num *rpc.BlockNumber) (*reward.StakingInfo, error) ` return format to `map[string]interface{}` to change return key field name?

The reason i wanna change this is that i need to use that field in another project.
So, if the field name needs to be changed, i prefer to use the changed name (for less future maintenance).
If we did not intentionally rename the field, and we do not plan to change it in the future, i can use the `PoCAddr` field name as is in other projects.

Reply:
If old data is the only thing we cannot change,
- perhaps we can add json tag to StakingInfo (might affect `GetStakingInfo` API)
```
KGFAddr  common.Address `json:"PoCAddr"`
```

- or fixup at `getStakingInfoFromDB()` (unnoticeable from other modules)
```
stakingInfoDb := &struct {
    StakingInfo
    PoCAddr common.Address
}{}
json.Unmarshal(jsonByte, stakingInfoDb)

stakingInfo := stakingInfoDb.StakingInfo
stakingInfo.KGFAddr = stakingInfoDb.PoCAddr
```